modules/core/src/v2/coins/btc.ts

```diff
@@ -67,17 +67,19 @@ export class Btc extends AbstractUtxoCoin {
   }
 
   recoveryBlockchainExplorerUrl(url: string): string {
-    // TODO: BG-23161 - replace smartbit block explorer which is now permanently down
-    throw new Error('btc recovery under maintenance');
+    return common.Environments[this.bitgo.getEnv()].blockchairBaseUrl + url;
   }
 
+  // using blockchair api: https://blockchair.com/api/docs#link_300
+  // https://api.blockchair.com/{:btc_chain}/dashboards/address/{:address}₀
```

Comment:
that's more of an implementation comment than a jsdoc and should therefore live inside the func (where you make the request)

Reply:
the commit message could also use a reference to the docs

Reply:
fixed